projects/observability/src/shared/components/gauge-list/gauge-list.component.test.ts

```diff
@@ -1,5 +1,5 @@
 import { FormattingModule } from '@hypertrace/common';
-import { GaugeListComponent, SortNData } from '@hypertrace/observability';
+import { GaugeItem, GaugeListComponent } from '@hypertrace/observability';
```

Comment:
bad import, should be relative.

Reply:
Interesting. I guess I will have to fix WebStorm

Reply:
The auto suggested import always suggests the aliased imports